analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs

```diff
@@ -84,6 +88,14 @@ language switch
             _ => throw new UnexpectedLanguageException(language)
         };
 
+    public bool AnalyzeRazorCode(string language) =>
```

Comment:
I think the name is misleading and it should, in fact, be `IsRazorAnalysisEnabled`. 

`AnalyzeRazorCode` could be read as a method that does the actual analysis and returns a value indicating if the operation succeded or not.